packages/vitest/src/node/config.ts

```diff
@@ -113,6 +113,9 @@ export function resolveConfig(
     }
   }
 
+  if (resolved.coverage.enabled && resolved.coverage.provider === 'c8' && resolved.browser.enabled)
```

Comment:
It's possible to run tests in a browser only for a subset of files. That's why there is a `ctx.isBrowserEnabled`

Reply:
Do you mean a case where the `poolMatchGlobs` would be used?

Should we in those cases print a warning message that `c8` is simply ignore in browser tests and still run the tests. In Node tests it would be enabled as before.

Reply:
Yes, I mean both. `browser` can be disabled, but might still have browser tests in `poolMatchGlobs`, and vice versa.

Reply:
The `poolMatchGlobs` is now detected in [ae63ed0](https://github.com/vitest-dev/vitest/pull/3040/commits/ae63ed0865e70f7742fa51f3ce6b0a62a7f15174).